data/pools/transactionPool.go

```diff
@@ -771,6 +771,26 @@ func (pool *TransactionPool) recomputeBlockEvaluator(committedTxIds map[transact
 	return
 }
 
+func (pool *TransactionPool) getStateProofStats(txib transactions.SignedTxnInBlock, encodedLen int) (stateProofStats telemetryspec.StateProofStats) {
```

Comment:
transactions.SignedTxnInBlock is a pretty big struct. I don't see any reason why not to pass it by pointer